rafthttp/peer.go

```diff
@@ -76,8 +77,8 @@ type Peer interface {
 
 // peer is the representative of a remote raft node. Local raft node sends
 // messages to the remote through peer.
-// Each peer has two underlying mechanisms to send out a message: stream and
-// pipeline.
+// Each peer has three underlying mechanisms to send out a message: stream,
+// pipeline and snapshot sender.
```

Comment:
I don't understand this one. snapshot sender is the way to send out snapshot message.